frontend/src/app/services/model/model.service.ts

```diff
@@ -76,6 +76,12 @@ export class ModelService {
     );
   }
 
+  deleteModel(project_slug: string, model: Model): Observable<any> {
+    return this.http.delete(
```

Comment:
```suggestion
  deleteModel(project_slug: string, model: Model): Observable<void> {
    return this.http.delete<void>(
```
Since we do not expect a return value, I would use `void` here.